tests/components/history_stats/test_sensor.py

```diff
@@ -1530,23 +1637,7 @@ def _fake_states(*args, **kwargs):
     # and will see that the sensor is ON starting from midnight.
     t3 = t2 + timedelta(minutes=1)
 
-    def _fake_states_t3(*args, **kwargs):
-        return {
-            "binary_sensor.state": [
-                ha.State(
-                    "binary_sensor.state",
-                    "on",
-                    last_changed=t3.replace(hour=0, minute=0, second=0, microsecond=0),
-                    last_updated=t3.replace(hour=0, minute=0, second=0, microsecond=0),
-                ),
-            ]
-        }
-
     with (
-        patch(
```

Comment:
Rolling over to a new window no longer queries the recorder.

Reply:
I guess now almost every typical usage of history stats will hit the recorder once on bootup and never again. Hope this works 😬 😅